lib/stat_tracker.rb

```diff
@@ -10,24 +10,15 @@ def initialize(games, teams, game_teams)
   end
 
   def self.from_csv(locations)
-    games_path = get_games(locations[:games])
-    teams_path = get_teams(locations[:teams])
+    games = get_games(locations[:games])
+    teams = get_teams(locations[:teams])
     game_teams = get_game_teams(locations[:game_teams])
-    StatTracker.new(games_path, teams_path, game_teams)
-  end
-
-  def self.get_games(games_path)
-
-  end
-
-  def self.get_teams(teams_path)
-
+    StatTracker.new(games, teams, game_teams)
   end
 
   def self.get_game_teams(path)
-    rows = CSV.read(path, headers: true)
     game_teams = {}
-    rows.each do |row|
+    CSV.foreach(path, headers: true) do |row|
       game_teams["#{row[0]}-#{row[1]}".to_sym] = GameTeam.new(row)
```

Comment:
Looks like getting your key for your game_teams hash was a bit tricky. Great job on this!! :-)